examples/README.md

```diff
@@ -0,0 +1,61 @@
+﻿# Common switches
+
+OpenThread allows you to [configure](https://openthread.io/guides/build?hl=en#configuration) the stack for different functionality and behavior. This configuration is based on changing compile-time constants during the build process using common switches listed in `/examples/common-switches.mk`.
+
+This page lists the available common switches with description and additional information. For build command examples, see [build examples](https://openthread.io/guides/build?hl=en#build_examples).
+
+| Makefile switch | CMake switch | Default value | Description | Additional information |
```

Comment:
Thanks for putting this together @piotrkoziar, great to finally have a reference of all the switches.  I have a major suggestion for the format: the table is a bit wide because some flags are long and requires horizontal scrolling to view, we should try to eliminate that for readability.  I would suggest a few things:

1. Combine the Description and Additional information columns, the Description is often repeated in the Additional Info, and Additional Info is blank half the time anyway
1. Drop the "Default value" column since nearly all of them default to 0... instead note at the top that all switches default to 0 unless otherwise noted (which we could do in the Description column)
1. Don't spell out a full link like https://openthread.io/guides/build/features/child-supervision... link the inline text directly

If you don't have the time to reformat, we can do it in a later pass, but I think condensing the columns would work well.

Alternatively, you can try adding `<wbr>` tags in the longer switch strings to force line breaks and reduce the column widths.